frontend/src/cards/ui/GeoContext.tsx

```diff
@@ -45,10 +53,23 @@ export default function GeoContext(props: GeoContextProps) {
 }
 
 export function getPopulationPhrase(
-  populationQueryResponse: MetricQueryResponse
+  populationQueryResponse: MetricQueryResponse,
+  demographicType: DemographicType,
+  metricId: MetricId
 ): string {
-  const totalPopulation: string =
-    populationQueryResponse.data?.[0].population?.toLocaleString() ?? 'unknown'
-
-  return `Total Population: ${totalPopulation}`
+  if (metricId === 'population') {
+    const popAllCount: string =
+      populationQueryResponse.data?.[0].population.toLocaleString()
+    console.log(popAllCount)
+    return `Total Population: ${popAllCount}`
+  }
+  if (PHRMA_METRICS.includes(metricId)) {
+    const allRow = populationQueryResponse.data?.find(
+      (row) => row[demographicType] === ALL
+    )
+    const popAllCount: string =
+      allRow?.phrma_population.toLocaleString() ?? 'unknown'
+    return `Total Medicare Population: ${popAllCount}`
```

Comment:
Eventually, I'd like to add this to the metricConfig to determine the population phrase so that it's not hardcoded.